.kitchen.dokken.yml

```diff
@@ -1,76 +1,75 @@
 driver:
   name: dokken
   privileged: true # because Docker and SystemD/Upstart
-  chef_version: current
+  chef_version: <%= ENV['CHEF_VERSION'] || 'current' %>
 
 transport:
   name: dokken
 
 provisioner:
   name: dokken
+  deprecations_as_errors: true
 
 verifier:
   name: inspec
 
 platforms:
+- name: amazonlinux
+  driver:
+    image: dokken/amazonlinux
+    pid_one_command: /sbin/init
+
 - name: debian-7
   driver:
-    image: debian:7
+    image: dokken/debian-7
```

Comment:
It's still supported for a while. Also didn't you say you were logging off for the night?